config/config.go

```diff
@@ -635,7 +641,9 @@ type P2PConfig struct { //nolint: maligned
 	PexReactor bool `mapstructure:"pex"`
 
 	// Set true to enable self kill when there are no peers
-	SelfKillNoPeers bool `mapstructure:"self-kill-no-peers"`
+	// DEPRECATED: use SelfRemediationConfigs instead
+	// To be removed post mainnet
+	DeprecatedSelfKillNoPeers bool `mapstructure:"self-kill-no-peers"`
```

Comment:
why dont we just remove it?

Reply:
yeah, good point let me just remove it. Would be easier to do so now than mainnet